script.js

```diff
@@ -0,0 +1,178 @@
+const quizQuestions = [
+  {
+    id: 1,
+    question: "What is the book sent to the Muslims called?",
+    options: [
+      { text: "Sunnah", isCorrect: false },
+      { text: "Quran", isCorrect: true },
+      { text: "Hadith", isCorrect: false },
+      { text: "Wudhu", isCorrect: false },
+    ],
+    explanation: "The book that is sent to the Muslims is called Quran.",
+  },
+  {
+    id: 2,
+    question: "What is the name of the last prophet in Islam?",
+    options: [
+      { text: "Musa", isCorrect: false },
+      { text: "Issa", isCorrect: false },
+      { text: "Muhammad", isCorrect: true },
+      { text: "Ibrahim", isCorrect: false },
+    ],
+    explanation: "The last prophet in Islam is Prophet Muhammad (peace be upon him).",
+  },
+  {
+    id: 3,
+    question: "What is the primary place of worship for Muslims?",
+    options: [
+      { text: "Church", isCorrect: false },
+      { text: "Synagog", isCorrect: false },
+      { text: "Mosque", isCorrect: true },
+      { text: "Temple", isCorrect: false },
+    ],
+    explanation: "The primary place of worship for Muslims is a mosque.",
+  },
+];
+
+let player1Score = 0;
+let player2Score = 0;
+let currentPlayer = 1;
+let currentQuestionIndex = 0;
+
+const questionTitle = document.querySelector('.questionTitle');
+const optionsContainer = document.querySelector('.options');
+const form = document.querySelector('#quizForm');
+const questionList = document.querySelector('.questionList');
+const player1ScoreHtml = document.getElementById('player1-score');
+const player2ScoreHtml = document.getElementById('player2-score');
+const currentTurnHtml = document.getElementById('current-turn');
+const playerForm = document.getElementById('playerForm');
+const player1UI = document.getElementById('player1UI');
+const player2UI = document.getElementById('player2UI');
+
+
+playerForm.addEventListener("submit", function(event){
+  event.preventDefault();
+  const player1ValueName = document.getElementById('player1ValueName').value;
+  const player2ValueName = document.getElementById('player2ValueName').value;
+
+  player1UI.textContent = player1ValueName
+  player2UI.textContent = player2ValueName
```

Comment:
I see that you display the names, but the assignment requested that you also add buttons to change their scores. You implemented something more than what the homework asked. You instead have to display the player names, and add buttons to simulate that they got an answer right or wrong, and just increase the scored accordingly.